tests/unittests/graphOptzTest.cpp

```diff
@@ -1061,6 +1061,42 @@ TEST_F(GraphOptz, sinkTransposeBelowConcatWithPredicate) {
   EXPECT_EQ(F_->getNodes().size(), 3);
 }
 
+TEST_F(GraphOptz, sinkTransposeBelowPad) {
+  // Before optims
```

Comment:
// The shape of the graph before the optimizations.

Reply:
Done.